app/Console/Commands/Init.php

```diff
@@ -7,20 +7,29 @@
 use Illuminate\Support\Carbon;
 use Illuminate\Support\Str;
 
-/**
- * @codeCoverageIgnore
- */
 class Init extends Command
 {
+    const TEMPLATES_PATH = '.templates';
+
     protected $signature = 'init {application-name : The application name }';
 
     protected $description = 'Initialize required project parameters to run DEV environment';
 
+    protected array $adminCredentials = [];
+
+    protected string $appUrl;
+
+    protected array $emptyValuesList = [];
+
+    protected string $readmeContent = '';
+
     public function handle(): void
     {
         $appName = $this->argument('application-name');
         $kebabName = Str::kebab($appName);
 
+        $this->appUrl = $this->ask('Please enter an application URL', 'http://localhost');
```

Comment:
the application URL may be generated from the `application-name` param, please set default value for the answer as `https://api.dev.{$kebabName}.com`

Reply:
Done